server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java

```diff
@@ -60,22 +60,23 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException {
         String pathInfo = req.getPathInfo();
-        if (Strings.isNullOrEmpty(pathInfo) || pathInfo.equals(ROOT_URL)) {
+        try {
+            download(getMailboxSession(req), DownloadPath.from(pathInfo), resp);
+        } catch (IllegalArgumentException e) {
+            LOGGER.error(String.format("Bad path '%s'", pathInfo), e);
```

Comment:
how can you be sure it's a bad path ? IllegalArgumentException could be thrown by another method